include/openssl/ec.h

```diff
@@ -852,7 +852,7 @@ EC_KEY *EC_KEY_copy(EC_KEY *dst, const EC_KEY *src);
  *  \param  src  the source EC_KEY object
  *  \return newly created EC_KEY object or NULL if an error occurred.
  */
-EC_KEY *EC_KEY_dup(const EC_KEY *src);
+DECLARE_ASN1_DUP_FUNCTION(EC_KEY)
```

Comment:
I double checked, I cannot see anything that makes this a ASN.1 thing.  Had it been about ECPKParameters or ECParameters, it would be a different story...

Reply:
Ah, sorry, I overlooked the first half of the "no" responses earlier today because they were automatically folded by this git page.
I've just reverted this and four further similar cases, such that now all of them should be covered.